src/population.hpp

```diff
@@ -29,15 +29,15 @@ namespace sonata {
 
 namespace {
 
-const char* H5_DYNAMICS_PARAMS = "dynamics_params";
-const char* H5_LIBRARY = "@library";
+const char* const H5_DYNAMICS_PARAMS = "dynamics_params";
+const char* const H5_LIBRARY = "@library";
```

Comment:
On the pedantic level of things: This looks like it might be a `constexpr [const] char [const] * const`.